src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx

```diff
@@ -65,9 +65,17 @@ const defaultRule: UnsavedIssueAlertRule = {
   environment: ALL_ENVIRONMENTS_KEY,
 };
 
+const POLLING_MAX_TIME_LIMIT = 3 * 60000;
+
 // TODO(ts): I can't get this to work if I'm specific -- should be: 'condition' | 'action';
 type ConditionOrAction = string;
 
+type RuleTaskResponse = {
+  status: string;
```

Comment:
@MeredithAnya Status should probably be typed as `'pending' | 'failed' | 'success'`